### ISSUE TITLE
Enhance search capabilities (they are very bad) 

### ISSUE THREAD

**Issue by skaldk** [bitwarden/mobile#1218]
I always name my Bitwarden credentials the same :
SERVICE NAME + login + login initials 

Let's say I have a johndoe@gmail.com account. 

In Bitwarden this entries will be  named as 
"GMAIL johndoe jd". 
Easy. 

If I search for "GMAIL _jo_..." it works fine. 

But... if I ever want to search "GMAIL _jd_", then it fail to find any match, like there is no entry with "GMAIL" and "jd" in its name.

It feels useless and frustrating. 

Also it would be useful to be able to sort search result by domain, entry name, or login. 

When you are looking for GOOGLE or GMAIL in a credential list, you may search your credentials for one of your Gmail account, or your credentials for any other site using your @gmail adress. 
Sorting in a way or another seems an obvious stuff to do, but I see no option to do that. 

So in short, sorting stuff and better search would help a lot.

**Comment by cscharf** [comment]
Hi @skaldk , thank you for your feedback, however we use GitHub issues as a place to track bugs and other development related issues. The [Bitwarden Community Forums](https://community.bitwarden.com) has a section for submitting, voting for, and discussing product [feature requests](https://community.bitwarden.com/c/feature-requests) like this one.

Please [sign up on our forums](https://community.bitwarden.com/signup) and search to see if this request already exists. If so, you can vote for it and contribute to any discussions about it. If not, you can re-create the request there so that it can be properly tracked.

This issue will now be closed. Thanks!

**Comment by skaldk** [comment]
Haa sorry. Didn't know exactly where to post this tbh...
